### PR TITLE
Double/Triple bracket matching

### PR DESCRIPTION
**Pull request by Anduh**
- Bracket identification: wikitext templates (`{{ }}`), template parameters (`{{{ }}}`) are separatly identified from `{ }`, and will be grouped together when selected
- if `editor.bracketPairColorization.enabled` is used, highlights make sense

**Before**
vscode-wikitext-before

**After**
vscode-wikitext-after

## Other
- my `yarn.lock` & `package-lock.json` had bunch of changes, but excluded them as irrelevant

### CODE REVIEW

**Review by Frederisk — CHANGES_REQUESTED** [review]
The is also a part of `tmLanguage` file that needs to be modified, this part may be completed by me.

File: language-configuration.yaml [hunk 1]
```diff
@@ -19,6 +19,10 @@ brackets:
     - ">"
   - - "-{"
     - "}-"
+  - - "{{{"
+    - "}}}"
+  - - "{{"
+    - "}}"
   - - "{"
     - "}"
```
[the remark this answers is not in the document]
Comment (Anduh) [reply language-configuration.yaml:27]: > should `{}` be deleted?

Kinda feel it should, not aware of actual use of it. If we remove it, we need to add double/triple to the auto-closing part

[the remark this answers is not in the document]
Comment (Frederisk) [reply language-configuration.yaml:27]: Delete it, this is how the brackets worked in the past.

**Comment by Frederisk** [comment]
@Anduh, Is there any progress on the change requested?

**Comment by Anduh** [comment]
Oh right, I tested additional changes some day ago, but also ended up adding a bunch of snippets, just didn't clean up things to submit as separate PR yet.

I'll update this PR by tomorrow.

**Comment by Frederisk** [comment]
Continue #52.

**Pull request closed without merging**